.github/workflows/release_notifications.yml

```diff
@@ -0,0 +1,178 @@
+# description of this workflow, can be anything you want
+name: Send Release Notifcations
+
+# we need to let GitHub know _when_ we want to release, typically only when we create a new tag.
+# this will target only tags, and not all pushes to the master branch.
+# this part can be heavily customized to your liking, like targeting only tags that match a certain word,
+# other branches or even pullrequests.
+# 'workflow_dispatch' allows us to re-run a failed job.
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+# a workflow is built up as jobs, and within these jobs are steps
+jobs:
+
+## Note: this is slightly messy, there might be a better way to do this, but im not sure how.
```

Comment:
You could create a setup step and use outputs to transfer this to the other steps. You can even define the other jobs to only run if your setup step ran succesfully using the "needs" qualifier.
see https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs